### PR TITLE
Small fixes to transpile Monica_SoilTemp Crop2ML code into all other frameworks and languages

### PR DESCRIPTION
**Pull request by bergm**
Hello Cyrille,

I finally managed to look into the code and fix the small issue to compile our Crop2ML code into the other languages. This shouldn't break anything. But who knows. :-)

Cheers, 
Michael

### CODE REVIEW

**Comment by pradal** [comment]
Thanks Michael,
We will review it soon before merging!
Thanks

File: src/pycropml/code2nbk.py [hunk 1]
```diff
@@ -48,24 +48,25 @@ def generate_nb(self, language, tg_rep, namep, mc_name=None):
         var = ["Auxiliary", "Rate", "State", "Exogenous"]
         if language in ("cs", "java"):
             for v in var:
-                fileVar = Path(os.path.join(tg_rep, "%s%s.%s" % (mc_name.capitalize() if language=="java" else mc_name, v, language)))
+                fileVar = Path(os.path.join(tg_rep, f"{mc_name}{v}.{language}"))
```
Comment (pradal) [inline src/pycropml/code2nbk.py:51]: Add capitalize for java:

   ```
_name = mc_name.capitalize() if language=="java" else mc_name
fileVar = Path(os.path.join(tg_rep, f"{mc_name}{v}.{language}"))
```

File: src/pycropml/render_cyml.py [hunk 2]
```diff
@@ -245,7 +245,7 @@ def generate_test(self, model_unit):
 
                     ins = inouts['inputs']
                     outs = inouts['outputs']
-                    print(outs.keys())
+                    #print(outs.keys())
```
Comment (pradal) [inline src/pycropml/render_cyml.py:248]: Nice to remove print, thks

**Review by pradal — CHANGES_REQUESTED** [review]
Excellent.
There is just one case (with java) that have to be managed.
Otherelse, everything is OK

File: src/pycropml/code2nbk.py (hunk 1 above)
Comment (pradal) [reply src/pycropml/code2nbk.py:51]: Thanks for the answer.
This is surely an old fix that may be also fixed somewhere else.

**Review by pradal — APPROVED** [review]

File: src/pycropml/code2nbk.py (hunk 1 above)
Comment (bergm) [reply src/pycropml/code2nbk.py:51]: Ok. I hope so. Else, I will fix it when any problem appears. :-)

**Pull request merged**